language/move-prover/bytecode/src/stackless_bytecode.rs

```diff
@@ -226,6 +226,38 @@ impl BorrowNode {
     }
 }
 
+/// A borrow edge with a known offset -- used in memory operations
```

Comment:
Please move the type declarations to the top of the file where the others are (Bytecode, Operation). Boilerplate impls (like for Display) should go towards the end,